packages/logger/README.md

```diff
@@ -41,10 +41,59 @@ npm install @sapphire/plugin-logger @sapphire/framework
 
 ## Usage
 
+This registers the necessary options and methods in the Sapphire client to be able to use the log plugin.
+
 ```typescript
+// Main bot file
+// Be sure to register the plugin before logging in with the bot.
 import '@sapphire/plugin-logger/register';
 ```
 
+To be able to use the Logger anywhere you first need to import the `container` property from the `@Sapphire/framework` package.
+
+```typescript
+import { container } from '@sapphire/plugin-logger';
+
+export class MyAwesomeService {
+	public printAwesomeLog() {
+		container.logger.info('log message');
+	}
+}
+```
+
+In pieces (commands, arguments, preconditions, etc.), the container is already available on the class as `this.container`. Therefore there is often no need to specifically import the container in such files to access the Logger methods.
```

Comment:
```suggestion
Note that in pieces (commands, arguments, preconditions, etc.), the container is already available on the class as `this.container`, so there is typically no need to import the container to access the Logger methods from them.
```

Reply:
As an aside, this should likely be above the prior example, starting with "In pieces". After that, you can prefix the example with "Elsewhere", instead of implying the container must be imported for use **anywhere** and then contradicting it with this note.